sdk/search/Azure.Search.Documents/tests/SearchIndexerClientTests.cs

```diff
@@ -13,6 +13,7 @@
 
 namespace Azure.Search.Documents.Tests
 {
+    [ClientTestFixture(SearchClientOptions.ServiceVersion.V2020_06_30, SearchClientOptions.ServiceVersion.V2020_06_30_Preview)]
```

Comment:
Can we move this to [SearchTestBase](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/search/Azure.Search.Documents/tests/Utilities/SearchTestBase.cs#L25) instead?

Reply:
I'll do that in a subsequent PR to handle any fallout.